tensorboard/webapp/widgets/data_table/column_selector_component.ts

```diff
@@ -45,6 +45,7 @@ export class ColumnSelectorComponent implements OnInit, AfterViewInit {
 
   searchInput = '';
   selectedIndex$ = new BehaviorSubject(0);
+  isActive = false;
```

Comment:
Given that you've written the accessor function you might as well make this `private`
```suggestion
  private isActive = false;
```